gbfs.md

```diff
@@ -273,9 +273,17 @@ Field Name | Required | Type | Defines
+\- `seating_capacity` | Optional | Non-negative integer | The number of riders the vehicle can legally accommodate.
+\- `share_type` | Optional | Enum | The sharing model of the vehicle.<br /><br />Current valid values are:<br /><br /><ul><li>`roundtrip` _(vehicle must be returned to pickup location)_</li><li>`one-way` _(vehicle can be returned to any station in the service area)_</li><li>`free_floating` _(vehicle can be returned anywhere in the service area)_
+\- `propulsion_type` | Yes | Enum | The primary propulsion type of the vehicle. <br /><br />Current valid values are:<br /><ul><li>`human` _(Pedal or foot propulsion)_</li><li>`electric_assist` _(Provides power only alongside human propulsion)_</li><li>`electric` _(Contains throttle mode with a battery-powered motor)_</li><li>`combustion` _(Contains throttle mode with a gas engine-powered motor)_</li></li>`combustion_diesel` _(Contains throttle mode with a diesel engine-powered motor)_</li><li>`hyrbid` _(Contains throttle mode with a mixed gas engine-powered and battery-powered motor)_</li><li>`hybrid_diesel` _(Contains throttle mode with a mixed diesel engine-powered and battery-powered motor)_<br /><br />This field was inspired by, but differs from the propulsion types field described in the [Open Mobility Foundation Mobility Data Specification](https://github.com/openmobilityfoundation/mobility-data-specification/blob/master/provider/README.md#propulsion-types).
+\- `min_fuel_level` | Optional | Enum | Minimum amount of fuel that must be left in the vehicle at the end of a trip.<br /><br />Current valid values are:<br /><br /><ul><li>`quarter_full`</li><li>`half_full`</li><li>`three_quarters_full`</li><li>`full`</li><li>`pickup_level` _(Same amount of fuel as at start of trip)_
 \- `max_range_meters` | Conditionally Required | Non-negative float | If the vehicle has a motor (as indicated by having a value other than `human` in the `propulsion_type` field), this field is required. This represents the furthest distance in meters that the vehicle can travel without recharging or refueling when it has the maximum amount of energy potential (for example, a full battery or full tank of gas).
 \- `name` | Optional | String | The public name of this vehicle type.
+\- `vehicle_accessories` | Optional | Array | Descriptions of accessories available in the vehicle. <br /><br />Current valid values are:<br /><br /><ul><li>`baby_seat`</li><li>`child_seat`</li><li>`winter_tires`
```

Comment:
Not to sure if this is too specific about cars again, I could image extending our own scooter and bike feeds with this field, specifying for example if the vehicle has a bell, how many brakes, bicycle basket, etc. Maybe this should just be part of the `vehicle_desc` field instead?